src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
Also, copying the project can add a lot to the page's footprint, since there's now twice as many models and event listeners as before, along with all of the Leaflet maps.